go.mod

```diff
@@ -53,14 +53,14 @@ require (
 	k8s.io/component-base v0.27.5
 	k8s.io/component-helpers v0.27.5
 	k8s.io/helm v2.16.1+incompatible
```

Comment:
We should drop the old helm throughout the repo and only use the new helm. Using two helm versions of two different major versions can cause incompatibility issues.

Reply:
As an option we can place needed functionality locally.